hsctl

```diff
@@ -102,6 +103,10 @@ preflight_hs() {
             echo "*** creating directory: ${HOST_SSL_DIR} ***"
             mkdir ${HOST_SSL_DIR};
         fi
+        if [[ ! -d ${HS_LOG_FILES} ]]; then
+            echo "*** creating directory: ${HS_LOG_FILES} ***"
+            mkdir ${HS_LOG_FILES};
```

Comment:
Interesting, I've tended to use `-d` for directories, or add `-L` to verify it's not a symlink but I can change to be `-p` if that is a better practice.